src/content/en/2021/jamstack.md

```diff
@@ -1,20 +1,506 @@
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+featured_quote: Jamstack young and fast growing technology to build modern websites. That what people say and that's proved by numbers.
+featured_stat_1: 1.1%
+featured_stat_label_1: Jamstack websites adoption among web
+featured_stat_2: 19%
+featured_stat_label_2: Median value of JQuery usage among Jamstack sites
+featured_stat_3: 1.7MB
+featured_stat_label_3: Median value of resource wights among Jamstack sites
 ---
 
-## TODO
+## Introduction
+
+Jamstack has revolutionized the way we think about building for the web by providing a simpler developer experience, better performance, lower cost and greater scalability - [more](https://jamstack.wtf/). Jamstack stands on JavaScript, API, and Markup architecture. These 3 foundations are decoupled and the Jamstack site can be built purely using markup. Using pure HTML - is “kinda” Jamstack, but it’s really hard to scale. Lucky for us, there's a huge ecosystem of Static Site Generators (SSGs).
+
+JavaScript based:
+* Next.js
+* Gatsby
+* Nuxt.js
+* etc
+
+Old-fashioned:
+* Eleventy
+* Hugo
+* Jekyll
+* Hexo
+* etc
+
+And many [more](https://jamstack.org/generators/).
+
+They allow building sites converted to “pure” HTML and JavaScript goodness if needed.
+
+For more complex sites, data has to be structured. There are several ways to store and manage data using [headless CMSs](https://jamstack.org/headless-cms/) via API.
+
+Moreover, Jamstack sites need support for server interactions. Like form submittions or user input processing. Services like Netlify provides [serverless functions](https://www.netlify.com/products/functions/) support.
+
+The goal of this chapter is to identify what’s the main SSGs used on Jamstack.Adoption of Jamstack technology year over year. Will answer how they are distributed around the world, what is the performance of Jamstack sites and how it is growing. Explore data of different CDN providers for Jamstack sites. Dive into results of resources used for Jamstack sites and their impact on user experiance.
+
+Worth to mention some data disclaimers:
+1. HTTPArchive data of detected SSGs is based on Wappalyzer technology. It can’t detect whether the site was built with Eleventy. Also, it can’t detect if the site was generated by Next.js [Static Rendering](https://nextjs.org/docs/basic-features/pages#static-generation-recommended) or [Server Side Rendering](https://nextjs.org/docs/basic-features/pages#server-side-rendering).
+2. In our analysis, we can’t get any info related to headless CMSs, hence we will not cover this part either.
+3. We visualise SSG data using top 5 used SSGs based on number of sites built with these SSGs.
+
+Most of the data is represented for mobile and desktop websites. There's a fully described [​​methodology](./methodology) used in HTTPArchive.
+
+## Adoption of SSGs
+
+### Ranking
```

Comment:
We avoid stacked headers like these. Every heading should have at least one sentence in it explaining what this section will cover.

There's a few instances of this so could you fix them all?